backend/src/core/entity/Group.ts

```diff
@@ -1,15 +1,39 @@
-import { type User } from "./User";
+import type { Result } from "src/utils";
 
 export class Group {
-  name: string;
-  members: User[];
+  /** Group's id */
+  public readonly id: number;
 
-  constructor(name: string, members: User[]) {
+  /** Group's name */
+  public name: string;
```

Comment:
Pourquoi avoir les attributs en Public ?

Reply:
Why not?

---

* C'est l'implémentation la plus simple, il faut bien pouvoir y accéder !
* On est pas en Java :stuck_out_tongue_closed_eyes: 

Pour `membersId` et `transactionsId`, ça pourrait s'entendre s'ils étaient en readonly à mon avis, mais bon c'est pas le cas ^^